### ISSUE TITLE
not getting anything Something port issue 

### ISSUE THREAD

**Issue by kedar2407** [AhMyth/AhMyth-Android-RAT#262]
i cant get anything from user with all settings and gui not working in command line installation also i have to connect both device on one network that i can see data but also iam able to only see contacts and call l0g please do something

**Comment by Morsmalleo** [comment]
Yes I dont know why this problem has occurred, I may not be the dev of the AhMyth Rat, but I can say this,

- the building failed problem is solved by installing Openjdk-8-jdk from Debian Stretch, in my fork of AhMyth, I have added a `AhMyth.sh` file for Debian and Kali Linux users to setup AhMyth completely with the correct version of openJDK, or you can just `dpkg -i` the openJDK-8-jdk.deb file located in `AhMyth/AhMyth-Server/` on my fork of AhMyth.

- The problem with the AhMyth browser not selecting Apk files to bind with is a problem in the code of the `AppCtrl.js` file which has also been fixed on my fork of AhMyth.

- The AhMyth Interface not working at all is solved by **NOT NPM INSTALLING electron-nightly**

Still yet to figure out why most of the actions in the Lab part of AhMyth dont work, because I could only access my contacts list as well, nothing else, I did see a comment on here somewhere where someone had everything working including all the options in the victims lab by installing AhMyth from source
